src/main/java/org/terasology/launcher/gui/javafx/ApplicationController.java

```diff
@@ -199,8 +199,8 @@ protected void openSettingsAction() {
                 root = fxmlLoader.load();
             }
 
-            final SettingsController settingsController = fxmlLoader.getController();
-            settingsController.initialize(launcherDirectory, downloadDirectory, launcherSettings, gameVersions, settingsStage);
+            //final SettingsController settingsController = fxmlLoader.getController();
```

Comment:
As mentioned above, please don't check in commented code. Either discard it or use a feature toggle.